### PR TITLE
Minor fixes + counterparts for getStringSet and putStringSet

### PR DESCRIPTION
**Pull request by linakis**
Fixed returned types to match actual type.

### CODE REVIEW

File: securestoragelibrary/src/main/java/de/adorsys/android/securestoragelibrary/SecurePreferences.java [hunk 1]
```diff
@@ -59,6 +62,16 @@ public static void setValue(@NonNull String key, int value,
         setValue(key, String.valueOf(value), context);
     }
 
+    @RequiresApi(Build.VERSION_CODES.JELLY_BEAN_MR2)
+    public static void setValue(@NonNull String key, Set<String> value,
+                                @NonNull Context context) throws CryptoException {
+        setValue(key + "_cnt", String.valueOf(value.size()), context);
```
Comment (luckyhandler) [inline securestoragelibrary/src/main/java/de/adorsys/android/securestoragelibrary/SecurePreferences.java]: Cnt = count? if yes, could you call it count, please? And put it into a global variable as you use it later again.

File: securestoragelibrary/src/main/java/de/adorsys/android/securestoragelibrary/SecurePreferences.java [hunk 2]
```diff
@@ -59,6 +62,16 @@ public static void setValue(@NonNull String key, int value,
...
+
+        int i = 0;
+        for(String s : value) {
```
Comment (luckyhandler) [inline securestoragelibrary/src/main/java/de/adorsys/android/securestoragelibrary/SecurePreferences.java]: please run auto format in AndroidStudio once

File: securestoragelibrary/src/main/java/de/adorsys/android/securestoragelibrary/SecurePreferences.java [hunk 3]
```diff
@@ -84,14 +97,29 @@ public static float getFloatValue(@NonNull String key, @NonNull Context context,
...
+    public static long getLongValue(@NonNull String key, @NonNull Context context, long defValue) {
         return Long.parseLong(getStringValue(key, context, String.valueOf(defValue)));
     }
 
-    public static float getIntValue(@NonNull String key, @NonNull Context context, int defValue) {
+    public static int getIntValue(@NonNull String key, @NonNull Context context, int defValue) {
         return Integer.parseInt(getStringValue(key, context, String.valueOf(defValue)));
     }
 
+    public static Set<String> getStringSetValue(@NonNull String key, @NonNull Context context, Set<String> defValue) {
```
Comment (luckyhandler) [inline securestoragelibrary/src/main/java/de/adorsys/android/securestoragelibrary/SecurePreferences.java]: add @NonNull annotation for the Set, please
add @NonNull for the return value as well

File: securestoragelibrary/src/main/java/de/adorsys/android/securestoragelibrary/SecurePreferences.java [hunk 4]
```diff
@@ -84,14 +97,29 @@ public static float getFloatValue(@NonNull String key, @NonNull Context context,
...
+        int size = getIntValue(key + "_cnt", context, -1);
```
Comment (luckyhandler) [inline securestoragelibrary/src/main/java/de/adorsys/android/securestoragelibrary/SecurePreferences.java]: use the global variable I requested you to create previously

File: securestoragelibrary/src/main/java/de/adorsys/android/securestoragelibrary/SecurePreferences.java (hunk 1 above)
Comment (linakis) [reply securestoragelibrary/src/main/java/de/adorsys/android/securestoragelibrary/SecurePreferences.java]: yes cnt is count, will add a static final for it.

**Pull request merged**